AmplifyTestCommon/Helpers/AmplifyAssertions.swift

```diff
@@ -0,0 +1,43 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import XCTest
+
+@testable import CwlPreconditionTesting
+
+public func XCTAssertThrowFatalError(_ expression: @escaping () -> Void,
```

Comment:
Is this different from https://developer.apple.com/documentation/xctest/1500795-xctassertthrowserror

Reply:
Yes, it is meant to look like those XCTest functions. This one catches fatal errors like the CwlPreconditionTesting package does.